app/javascript/components/pages/YourTrips.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+
+export default class YourTrips extends Component {
+  render() {
+    const { trips, user } = this.props;
+    let yourtrips = trips.filter((trip) => {
+      return trip.user_id === user.id;
+    });
+    return (
+      <div>
+        <h1>Your Trips</h1>
+        {trips &&
+          yourtrips.map((trip) => {
```

Comment:
this check seems like it'd need to happen sooner like on line 7 because if there are no trips the trips.filter() will throw an error